src/content/developers/docs/consensus-mechanisms/pow/mining/index.md

```diff
@@ -56,6 +56,12 @@ The following provides an overview of how transactions were mined in Ethereum pr
 
 Every transaction is mined (included in a new block and propagated for the first time) once, but executed and verified by every participant in the process of advancing the canonical EVM state. This highlights one of the central mantras of blockchain: **Don’t trust, verify**.
 
+## Ommer (Uncle) Blocks
+
+As block mining under proof-of-work was probabilistic, there exists a non-zero chance that two valid blocks may have been published at the same time due to network latency. In this case, the protocol still has to determine the longest (and hence most "valid") chain while still ensuring fairness towards the miners by partially rewarding the unincluded valid block proposed by the miner. This encourages further decentralization of the network as smaller miners, who might face greater latency, are still able to generate a return via [ommer](https://ethereum.org/en/glossary/#ommer) block rewards.
```

Comment:
Use past tense when describing historical features to avoid confusion

```suggestion
As block mining under proof-of-work was probabilistic, sometimes two valid blocks were published simultaneously due to network latency. In this case, the protocol had to determine the longest (and therefore most "valid") chain while ensuring fairness towards miners by partially rewarding the unincluded valid block proposed by the miner. This encouraged further decentralization of the network as smaller miners, who might face greater latency, could still generate returns via [ommer](/glossary/#ommer) block rewards.
```